### ISSUE TITLE
re: Gmail Checker Improved

### ISSUE THREAD

**Issue by rkumar** [matryer/bitbar-plugins#409]
FAO: (@axeloz)
 
I get a "can't locate Mozilla/CA.pm in @INC" Line 13.

Thanks.

**Comment by axeloz** [comment]
Hello, 

did you install the CA perl package?
http://search.cpan.org/~abh/Mozilla-CA-20160104/lib/Mozilla/CA.pm

Thanks

**Comment by rkumar** [comment]
I have installed the CA package. I have updated PERL5LIB to the location of CA. On the terminal, it connects, but GMAIL prevents the sign in attempt --

> Someone just tried to sign in to your Google Account xxxx@gmail.com from an app that doesn't meet modern security standards. Google stopped this sign-in attempt

However, from bitbar, it is still unable to locate CA.pm in @INC. I am unable to see the entire content of @INC on the bitbar. I used "cpanm Mozilla::CA' so it has installed in my local user, and not system.

I understand this is an issue with my instalation of CA.pm (I am new to perl). But if Gmail prevents sign-in, then how are others using this ?
